### PR TITLE
Remove JS syntax > ECMA 2018

### PR DESCRIPTION
**Pull request by Mugen87**
Related issue: #25168

**Description**

This PR removes all JS syntax > ECMA 2018 from the code base.

@sunag I see optional chaining was frequently used in node material classes. For now let's stick to the ternary operator or ordinary `if` statements. I've also removed the usage of static fields. These are now just added to the class namespace like in the core. Having static methods is fine, btw.

### CODE REVIEW

File: examples/jsm/nodes/core/NodeBuilder.js [hunk 1]
```diff
@@ -297,7 +297,7 @@ class NodeBuilder {
 
 	hasGeometryAttribute( name ) {
 
-		return this.geometry?.getAttribute( name ) !== undefined;
+		return this.geometry ? ( this.geometry.getAttribute( name ) !== undefined ) : undefined;
```
Comment (LeviPesin) [inline examples/jsm/nodes/core/NodeBuilder.js:300]: This should be `this.geometry && this.geometry.getAttribute( name ) !== undefined`.

Comment (LeviPesin) [reply examples/jsm/nodes/core/NodeBuilder.js:300]: > Meaning when `this.geometry` is undefined, the method used to return `undefined`.

I think `this.geometry?.getAttribute( name ) !== undefined` is `false` in this case, not `undefined`.

Comment (Mugen87) [reply examples/jsm/nodes/core/NodeBuilder.js:300]: You're right! I've actually misread the statement.

**Comment by sunag** [comment]
Ohh!! No problems, as soon as you merger this PR I will update this https://github.com/mrdoob/three.js/pull/25170

**Pull request merged**